docs/user/using-docker.md

```diff
@@ -6,31 +6,31 @@ Turbinia supports the use of Docker by allowing a Task to execute its command th
 ## Enabling Docker usage
 In order to enable this feature, please take the following steps. 
 1. Install the Docker daemon on the Worker's host machine. Please visit the Docker website for the [Installation Guide](https://docs.docker.com/install/).
```

Comment:
Step 1 can be removed as the docker cli is already installed on the worker and the docker compose file has a docker-in-docker container running to manage the containers.
https://github.com/google/turbinia/blob/master/docker/local/docker-compose-dind.yml